### ISSUE TITLE
[BUG] Blog "A deep dive into faster semantic sparse retrieval in OpenSearch 2.12" link break

### ISSUE THREAD

**Issue by zhichao-aws** [opensearch-project/project-website#3522]
### Describe the bug

In neural sparse blogs, there are links to reference other blog posts. (an [example](https://opensearch.org/blog/improving-document-retrieval-with-sparse-semantic-encoders), further reading section)

I just found the ref link for blog "A deep dive into faster semantic sparse retrieval in OpenSearch 2.12" breaks.
The URL https://opensearch.org/blog/A-deep-dive-into-faster-semantic-sparse-retrieval-in-OS-2.12 throw errors, but with a slash, https://opensearch.org/blog/A-deep-dive-into-faster-semantic-sparse-retrieval-in-OS-2.12/ works.

Is this by expected? I noticed other blogs  also work without the slash

### Expected behavior

The blog render correctly once click the blog link

### Screenshots

![image](https://github.com/user-attachments/assets/a2bfea62-a80b-4142-861a-c8e7b7aed3ad)

### To Reproduce

1. Go to this [blog](https://opensearch.org/blog/improving-document-retrieval-with-sparse-semantic-encoders), further reading section
2. Click on link for [A deep dive into faster semantic sparse retrieval in OpenSearch 2.12](https://opensearch.org/blog/A-deep-dive-into-faster-semantic-sparse-retrieval-in-OS-2.12)
3. See error

### Additional Details

**Plugins**
Please list all plugins currently enabled.

**Screenshots**

**Host/Environment (please complete the following information):**

**Additional context**

**Comment by zhichao-aws** [comment]
@pajuric The root cause may come from the server hosting the blogs. I don't have access to that server, so not sure if I can do more on this issue besides changing the URL to end with a slash "/"

**Comment by nateynateynate** [comment]
This seems to be resolved. Are you still seeing this behavior @zhichao-aws ?

**Comment by zhichao-aws** [comment]
@nateynateynate Yes. The url https://opensearch.org/blog/A-deep-dive-into-faster-semantic-sparse-retrieval-in-OS-2.12 still not works

**Comment by nateynateynate** [comment]
@zhichao-aws : What I meant to ask is if we're linking to it anywhere with that incorrect link.

**Comment by zhichao-aws** [comment]
Yes we're still linking to it in other neural sparse blogs. Here is a PR to change the links: https://github.com/opensearch-project/project-website/pull/3650
